cloudfront/wellcomelibrary.org/edge-lambda/src/catalogue.ts

```diff
@@ -0,0 +1,14 @@
+export type Work = {
+    type: 'Work' | 'Collection' | 'Section' | 'Series';
+    id: string;
+    title: string;
+};
+
+export type CatalogueResultsList<Result = Work> = {
+    type: 'ResultList';
+    totalResults: number;
+    results: Result[];
+    pageSize: number;
+    prevPage: string | undefined;
```

Comment:
```suggestion
    prevPage?: string
```

There is a slight difference if you are implementing the type. e.g.
```ts
type A = { a: string | undefined }
const { a: undefined } // valid
const {} // invalid

type B = { b?: string }
const { a: undefined } // valid
const {} // valid
```